src/components/DetailsCard/index.js

```diff
@@ -0,0 +1,32 @@
+import React from "react";
+import { GiConqueror, GiMugShot, GiPlayerNext, GiPlayerPrevious } from "react-icons/gi";
+import { RiGitRepositoryFill } from "react-icons/ri"
+import "./index.css"
+
+export const DetailsCard = ({user}) => {
+
+   const renderCard = () => {
```

Comment:
I like the way you took the function out of the return and then just used the function lower down, makes the code very readable.